clockwork.go

```diff
@@ -65,6 +79,30 @@ func (rc *realClock) Since(t time.Time) time.Duration {
 	return rc.Now().Sub(t)
 }
 
+func (rc *realClock) NewTimer(d time.Duration) Timer {
+	return &realTimer{time.NewTimer(d)}
+}
+
+func (rc *realClock) AfterFunc(d time.Duration, f func()) Timer {
+	return &realTimer{time.AfterFunc(d, f)}
+}
+
+type realTimer struct {
+	t *time.Timer
```

Comment:
This could be an anonymous struct. That way, you don't have to define the `Reset` and `Stop` methods below, only `C()`. See #8 for a similar implementation of tickers